### PR TITLE
Cleaned up css files

### PR DESCRIPTION
**Pull request by JackStouffer**
* ~~removed overqualifed css rules~~
* removed commented out old code
* changed `0px` and `0em` to `0`
* added some extra prefixed css rules
* made long lines under 80 characters

### CODE REVIEW

**Comment by JackStouffer** [comment]
This speeds up rendering time on Chrome by 76ms (I only did one test though).

**Comment by CyberShadow** [comment]
I have not measured any performance improvements at all. As such, I think dequalifying CSS rules provides no benefit, and worsens readability. It might also change behavior in style sheets that build upon style.css, such as forum.dlang.org's stylesheet.

**Comment by JackStouffer** [comment]
>I have not measured any performance improvements at all.

It may very per browser, but these rules are performing less searches than their original versions, as

```
element#selector
```

does two searches while

```
#selector
```

performs one, so it must be faster.

> As such, I think dequalifying CSS rules provides no benefit, and worsens readability. 

I understand that this goes down to personal preference, but for me overqualified rules are just noise, as 

```
div#content
```
doesn't give you anymore *useful* information than
```
#content
```

**Comment by CyberShadow** [comment]
> It may very per browser,

I tested with Chrome.

> so it must be faster

You don't know that. The browser might be optimizing this as there can be at most one ID per document anyway.

Also, if it's faster by an insignificant amount, then it has no practical gain.

> but for me overqualified rules are just noise

Maybe in that example. In some other cases, it can be informative, e.g. when the element is something other than a div or span.

**Comment by JackStouffer** [comment]
@CyberShadow Ok, so I removed all of my changes where the element being styled was not either a div or a span, where the class name did not include the type of element being styled, or where the rule was not an id. I think these changes help remove noise while keeping things unambiguous for most rules.

**Comment by CyberShadow** [comment]
You haven't addressed the main concern:

> It might also change behavior in style sheets that build upon style.css, such as forum.dlang.org's stylesheet.

These changes change CSS specificity and as such are potentially breaking changes. Please just leave them out.

**Comment by JackStouffer** [comment]
@CyberShadow fixed

File: css/print.css [hunk 1]
```diff
@@ -11,12 +11,21 @@ pre
...
+#copyright,
+#github-ribbon,
+#top,
+.page-contents
 {
 	display: none;
 }
 
-div#content
+#content
```
Comment (CyberShadow) [inline css/print.css]: Still not fixed

File: css/style.css [hunk 2]
```diff
@@ -184,7 +163,7 @@ table.download-compilers h3
     text-align: center;
 }
 
-div#download-choose
+#download-choose
```
Comment (CyberShadow) [inline css/style.css]: Ditto

File: css/style.css [hunk 3]
```diff
@@ -274,7 +255,7 @@ h3.download
     font-weight: bold;
 }
 
-div.summary, div.description, div.keyval
+.summary, .description, .keyval
```
Comment (CyberShadow) [inline css/style.css]: Ditto. I see more in the diff.

**Comment by JackStouffer** [comment]
Sorry about that. I tried using a GUI git tool and it seems most of my discards didn't take.

**Comment by JackStouffer** [comment]
Seems to have worked this time

**Comment by CyberShadow** [comment]
Thx!

**Pull request merged**